### PR TITLE
Extended theme preprocess fix.

### PR DESCRIPTION
**Pull request by goranmiric**
Issue: [#372](https://github.com/ymcatwincities/openy/issues/372)

- [x] All coding styles are fulfilled and there are no any issues reported by CodeSniffer CI.
- [x] All tests are running and there are no failed tests reported by CI.
- [x] [Documentation](https://github.com/ymcatwincities/openy/tree/8.x-1.x/docs) has been updated according to PR changes.
- [x] [Steps for review](https://github.com/ymcatwincities/openy/pull/94#issue-204580200) have been provided according to PR changes.
- [x] Make sure you've provided all necessary hook\_update\_N to [support upgrade path](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Upgrade%20path.md).
- [x] Make sure your git email is associated with account on drupal.org, otherwise you won't get commits there.
- [x] If you would like to get credits on drupal.org, [check documentation](https://github.com/ymcatwincities/openy/blob/8.x-1.x/docs/Development/Contributing.md#drupalorg-credits).

Thank you for your contribution!

### CODE REVIEW

File: themes/openy_themes/openy_rose/openy_rose.theme [hunk 1]
```diff
@@ -266,7 +266,7 @@ function openy_rose_page_attachments_alter(array &$page) {
  */
 function openy_rose_preprocess_page(array &$variables) {
   // Set custom theme variables for page when they are set.
-  $image_fields = theme_get_setting('oepny_rose_image_fields');
+  $image_fields = \Drupal::config('openy_rose.settings')->get('oepny_rose_image_fields');
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/openy_rose.theme]: I think there's a typo in the config name `oepny_rose_image_fields` should be `openy_rose_image_fields`.
1. This should be fixed in this function.
2. This should be fixed here as well https://github.com/ymcatwincities/openy/blob/5a2478bbeb896ca2a3abe9ac88daa76c4baf8a19/themes/openy_themes/openy_rose/config/install/openy_rose.settings.yml#L2.
3. Upgrade path should be provided:
- Get value of `oepny_rose_image_fields`
- Save it as `openy_rose_image_fields`
- Remove `oepny_rose_image_fields` property.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by Sanchiz** [comment]
I’m sorry for brining additional work and it will be pretty reasonable if you would like to get current PR merge and fix the issue with typo in config name in follow up :)

**Comment by alexschedrov** [comment]
@ddrozdik @goranmiric I’m sorry for brining additional work and it will be pretty reasonable if you would like to get current PR merge and fix the issue with typo in config name in follow up :)

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1221 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1221 |
| Installation process | http://install.openy-dev.ffwua.com/build1221/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1221/

**Comment by gundevel** [comment]
Behat test results file URL: http://ci.openymca.org/build1221/profiles/contrib/openy/build/reports/behat

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1232 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1232 |
| Installation process | http://install.openy-dev.ffwua.com/build1232/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1232/

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1233/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #373 of commit dd3191dee39ca9a818893173b48fdd863351b5dc automatically merged.
GitHub pull request #373 of commit dd3191dee39ca9a818893173b48fdd863351b5dc automatically merged.
GitHub pull request #373 of commit dd3191dee39ca9a818893173b48fdd863351b5dc automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git6563296872287975339.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git647602209396024546.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/373/merge^{commit} # timeout=10
 > git branch -a --contains 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a # timeout=10
 > git rev-parse remotes/origin/pr/373/merge^{commit} # timeout=10
Checking out Revision 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a (origin/pr/373/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson2443253924196557805.sh
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ git pull
Already up-to-date.
[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory6714530915204907278.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1233 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/build.yml -i /tmp/inventory2745156494841050770.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1233 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/www/build1233/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1233 -e drupal_folder=/var/www/build1233 -e site_url=http://openy-dev.ffwua.com/build1233 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
failed: [localhost] => {"failed": true, "parsed": false}
BECOME-SUCCESS-ogllmnfqoqbkgeesbgsrbkgjcrzpuezf
Traceback (most recent call last):
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 2012, in <module>
    main()
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 291, in main
    changed |= recursive_set_attributes(module, file_args['path'], follow, file_args)
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 132, in recursive_set_attributes
    changed |= module.set_fs_attributes_if_different(tmp_file_args, changed)
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 1207, in set_fs_attributes_if_different
    file_args['path'], file_args['mode'], changed
  File "/var/lib/jenkins/.ansible/tmp/ansible-tmp-1495202461.17-135152445386476/file", line 1045, in set_mode_if_different
    path_stat = os.lstat(path)
OSError: [Errno 2] No such file or directory: '/tmp/git3262018958368964875.credentials'


FATAL: all hosts have already failed -- aborting

PLAY RECAP ******************************************************************** 
           to retry, use: --limit @/var/lib/jenkins/reinstall.retry

localhost                  : ok=3    changed=0    unreachable=0    failed=1   

FATAL: command execution failed
hudson.AbortException: Ansible playbook execution failed
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:227)
	at org.jenkinsci.plugins.ansible.AnsiblePlaybookBuilder.perform(AnsiblePlaybookBuilder.java:200)
	at hudson.tasks.BuildStepCompatibilityLayer.perform(BuildStepCompatibilityLayer.java:75)
	at hudson.tasks.BuildStepMonitor$1.perform(BuildStepMonitor.java:20)
	at hudson.model.AbstractBuild$AbstractBuildExecution.perform(AbstractBuild.java:779)
	at hudson.model.Build$BuildExecution.build(Build.java:205)
	at hudson.model.Build$BuildExecution.doRun(Build.java:162)
	at hudson.model.AbstractBuild$AbstractBuildExecution.run(AbstractBuild.java:537)
	at hudson.model.Run.execute(Run.java:1741)
	at hudson.model.FreeStyleBuild.run(FreeStyleBuild.java:43)
	at hudson.model.ResourceController.execute(ResourceController.java:98)
	at hudson.model.Executor.run(Executor.java:381)
ERROR: Ansible playbook execution failed
```

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1234 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1234 |
| Installation process | http://install.openy-dev.ffwua.com/build1234/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1234/

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1235/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #373 of commit 2e904d16b34b2cd8ad92ae774f798edd67c0f40c automatically merged.
GitHub pull request #373 of commit 2e904d16b34b2cd8ad92ae774f798edd67c0f40c automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git4220031033538623637.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git5476896978631698110.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/373/merge^{commit} # timeout=10
 > git branch -a --contains a4ed402933a75681a77d44eed4862c8ef68e719d # timeout=10
 > git rev-parse remotes/origin/pr/373/merge^{commit} # timeout=10
Checking out Revision a4ed402933a75681a77d44eed4862c8ef68e719d (origin/pr/373/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a4ed402933a75681a77d44eed4862c8ef68e719d
 > git rev-list 8a7e1f3a9db3e1219dd2267e2b7ddd2e3e4c164a # timeout=10
First time build. Skipping changelog.
[workspace] $ /bin/sh -xe /tmp/hudson4461593762214201625.sh
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ git pull
Already up-to-date.
[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory7050444425425177194.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1235 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/build.yml -i /tmp/inventory5507590835860485254.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1235 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/www/build1235/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1235 -e drupal_folder=/var/www/build1235 -e site_url=http://openy-dev.ffwua.com/build1235 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local
Build was aborted
Aborted by anonymous
```

**Comment by Sanchiz** [comment]
retest this please

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============

--------------
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1237/

**Build Log**
*last 1000 lines*

 ```
GitHub pull request #373 of commit 2e904d16b34b2cd8ad92ae774f798edd67c0f40c automatically merged.
GitHub pull request #373 of commit 2e904d16b34b2cd8ad92ae774f798edd67c0f40c automatically merged.
GitHub pull request #373 of commit 2e904d16b34b2cd8ad92ae774f798edd67c0f40c automatically merged.
Building in workspace /var/lib/jenkins/jobs/PR_BUILDER/workspace
Cloning the remote Git repository
Using shallow clone
Cloning repository https://github.com/ymcatwincities/openy.git
 > git init /var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
 > git --version # timeout=10
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git7842767155818486011.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/heads/*:refs/remotes/origin/* --depth=1 # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
 > git config --add remote.origin.fetch +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git config remote.origin.url https://github.com/ymcatwincities/openy.git # timeout=10
Cleaning workspace
 > git rev-parse --verify HEAD # timeout=10
No valid HEAD. Skipping the resetting
 > git clean -fdx # timeout=10
Fetching upstream changes from https://github.com/ymcatwincities/openy.git
using .gitcredentials to set credentials
 > git config --local credential.helper store --file=/tmp/git1181904946819386453.credentials # timeout=10
 > git -c core.askpass=true fetch --tags --progress https://github.com/ymcatwincities/openy.git +refs/pull/*:refs/remotes/origin/pr/* # timeout=5
 > git config --local --remove-section credential # timeout=10
 > git rev-parse origin/pr/373/merge^{commit} # timeout=10
 > git branch -a --contains a4ed402933a75681a77d44eed4862c8ef68e719d # timeout=10
 > git rev-parse remotes/origin/pr/373/merge^{commit} # timeout=10
Checking out Revision a4ed402933a75681a77d44eed4862c8ef68e719d (origin/pr/373/merge)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f a4ed402933a75681a77d44eed4862c8ef68e719d
 > git rev-list a4ed402933a75681a77d44eed4862c8ef68e719d # timeout=10
[workspace] $ /bin/sh -xe /tmp/hudson6328350320688717798.sh
+ cd /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm
+ git pull
Already up-to-date.
[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/deploy_code.yml -i /tmp/inventory8568212536380757579.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1237 -e server_docroot_folder=/var/www -e deploy_type=local -e profile_source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/sourcecode -e source=/var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/docroot -e profile_destination=profiles/contrib/openy -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Generate commentinfo.md file] ************ 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Make sure folder is not exist] *********** 
ok: [localhost]

TASK: [../custom/cibox-code-deploy | Sync files into docroot] ***************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Sync OpenY profile] ********************** 
changed: [localhost]

TASK: [../custom/cibox-code-deploy | Folder permissions] ********************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=6    changed=4    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/lib/jenkins/jobs/PR_BUILDER/workspace/vm/cibox/jobs/build.yml -i /tmp/inventory2443251275451706853.ini -f 5 -e workspace=/var/lib/jenkins/jobs/PR_BUILDER/workspace -e build_number=1237 -e server_docroot_folder=/var/www -e build_folder_prefix=build -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Folder permissions] **************************************************** 
changed: [localhost]

PLAY RECAP ******************************************************************** 
localhost                  : ok=2    changed=1    unreachable=0    failed=0   

[workspace] $ ansible-playbook /var/www/build1237/reinstall.yml -f 5 -e php_env_vars=APP_ENV=dev -e mysql_user=root -e mysql_password=root -e mysql_db=drupal1237 -e drupal_folder=/var/www/build1237 -e site_url=http://openy-dev.ffwua.com/build1237 -e pp_environment=default -e run_reinstall=true -i localhost, --connection=local

PLAY [localhost] ************************************************************** 

GATHERING FACTS *************************************************************** 
ok: [localhost]

TASK: [Include specific variables per environment] **************************** 
ok: [localhost]

TASK: [Getting timestamp for backup] ****************************************** 
skipping: [localhost]

TASK: [Getting name for backup] *********************************************** 
skipping: [localhost]

TASK: [Creating backup folder for environment] ******************************** 
skipping: [localhost]

TASK: [Making regular backup from {{ local_backup_environment }} environment] *** 
skipping: [localhost]

TASK: [Make new backup as latest] ********************************************* 
skipping: [localhost]

TASK: [Set writable cache dir] ************************************************ 
ok: [localhost]

TASK: [Set writable tmp dir] ************************************************** 
ok: [localhost]

TASK: [Set writable backup dir] *********************************************** 
ok: [localhost]

TASK: [Set writable default dir] ********************************************** 
changed: [localhost]

TASK: [Set writable files dir] ************************************************ 
changed: [localhost]

TASK: [Set writable private dir] ********************************************** 
changed: [localhost]

TASK: [Start the MySQL service] *********************************************** 
ok: [localhost]

TASK: [Getting name for settings (WINDOWS)] *********************************** 
skipping: [localhost]

TASK: [Getting name for settings (UNIX)] ************************************** 
skipping: [localhost]

TASK: [Remove settings.php] *************************************************** 
skipping: [localhost]

TASK: [Creating settings.php] ************************************************* 
skipping: [localhost]

TASK: [Adding database variable to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add general pre-settings to settings.php] ****************************** 
skipping: [localhost]

TASK: [Add environment specific pre-settings] ********************************* 
skipping: [localhost] => (item={'status': True, 'name': '$settings["hash_salt"]', 'value': 'g1W4KZgWCeWh7Af9_gSb9D9HOiNaBdWPvQD-8x0yWpcgNpTBppaHR5VNGQBGvh2k8P8uWa1V8w'})
skipping: [localhost] => (item={'status': False, 'name': '$config_directories["staging"]', 'value': 'sites/default/config/staging'})
skipping: [localhost] => (item={'status': True, 'name': '$config_directories["sync"]', 'value': 'sites/default/config/sync'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["install_profile"]', 'value': u'openy'})
skipping: [localhost] => (item={'status': True, 'name': '$settings["file_public_path"]', 'value': u'sites/default/files'})
skipping: [localhost] => (item={'status': True, 'name': '$config["system.file"]["path.temporary"]', 'value': '/tmp'})

TASK: [Copy settings back (WINDOWS)] ****************************************** 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
skipping: [localhost]

TASK: [Creating drupal database] ********************************************** 
skipping: [localhost]

TASK: [Downloading latest backup to local environment] ************************ 
skipping: [localhost]

TASK: [Removing possible abandoned dump] ************************************** 
skipping: [localhost]

TASK: [Importing from remote backup] ****************************************** 
skipping: [localhost]

TASK: [Copy latest backup to build folder] ************************************ 
skipping: [localhost]

TASK: [Importing from local backup] ******************************************* 
skipping: [localhost]

TASK: [Remove local backup from build folder] ********************************* 
skipping: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
skipping: [localhost]

TASK: [Clear drush cache] ***************************************************** 
skipping: [localhost]

TASK: [Rebuilding drupal registry] ******************************************** 
skipping: [localhost]

TASK: [Clear Drupal cache] **************************************************** 
skipping: [localhost]

TASK: [Updating database] ***************************************************** 
skipping: [localhost]

TASK: [Apply pending entity schema updates] *********************************** 
skipping: [localhost]

TASK: [Download stage file proxy] ********************************************* 
skipping: [localhost]

TASK: [Enable stage file proxy] *********************************************** 
skipping: [localhost]

TASK: [Set configuration for stage file proxy] ******************************** 
skipping: [localhost]

TASK: [Set folder for stage file proxy] *************************************** 
skipping: [localhost]

TASK: [Updates listing] ******************************************************* 
skipping: [localhost]

TASK: [Droping drupal database] *********************************************** 
ok: [localhost]

TASK: [Creating drupal database] ********************************************** 
changed: [localhost]

TASK: [Remove settings.php] *************************************************** 
ok: [localhost]

TASK: [Build project from drupal-org.make] ************************************ 
Build was aborted
Aborted by anonymous
```

**Comment by Sanchiz** [comment]
retest this please

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://ci.openymca.org/build1245 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build1245 |
| Installation process | http://install.openy-dev.ffwua.com/build1245/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.openymca.org:8080/job/PR_BUILDER/1245/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://ci.openymca.org/build1245/profiles/contrib/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**